cmd/build/helmify/kustomize-for-helm.yaml

```diff
@@ -173,3 +173,19 @@ spec:
       control-plane: controller-manager
       gatekeeper.sh/operation: webhook
       gatekeeper.sh/system: "yes"
+---
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: critical-pods
+  namespace: system
+spec:
+  hard:
+    pods: HELMSUBST_RESOURCEQUOTA_POD_LIMIT
+  scopeSelector:
+    matchExpressions:
+    - operator: In
+      scopeName: PriorityClass
+      values:
+      - HELMSUBST_DEPLOYMENT_CONTROLLER_MANAGER_PRIORITY_CLASS_NAME
+      - HELMSUBST_DEPLOYMENT_AUDIT_PRIORITY_CLASS_NAME
```

Comment:
These two should also be replaced in replacements.go and they should be set to some default value in values.yaml and added to the chart readme file.

Reply:
Those were introduced in a previous change, and already have replacements and default values set.

Reply:
Hi @ritazh, please confirm if this is resolved. I'd like to be sure this makes it into the next release.

Reply:
@jalseth apologies for the delay on this. Yes this is resolved.